### ISSUE TITLE
Enabling formatter results an error

### ISSUE THREAD

**Issue by son1ss** [rszyma/vscode-kanata#20]
If i enable formatter in settings, when trying to save I get this error:

recursive use of an object detected which would lead to unsafe aliasing in rust

```
[Error - 10:40:59 PM] Request textDocument/formatting failed.
  Message: Request textDocument/formatting failed with message: unreachable
  Code: -32603 
[Error - 10:41:00 PM] Notification handler 'textDocument/didSave' failed with message: recursive use of an object detected which would lead to unsafe aliasing in rust
[Error - 10:41:00 PM] Request textDocument/formatting failed.
  Message: Request textDocument/formatting failed with message: recursive use of an object detected which would lead to unsafe aliasing in rust
  Code: -32603 
[Error - 10:41:00 PM] Notification handler 'textDocument/didSave' failed with message: recursive use of an object detected which would lead to unsafe aliasing in rust
```

**Comment by rszyma** [comment]
Can you send panic stack trace? It should logged in extension output (see https://stackoverflow.com/a/65938179/9611188)

**Comment by son1ss** [comment]
@rszyma 

```
panicked at src/formatter/defsrc_layout/mod.rs:175:38:
internal error: entered unreachable code

Stack:

Error
    at N.exports.__wbg_new_abda76e883ba8a5f (c:\Users\fomin\.vscode\extensions\rszyma.vscode-kanata-0.8.4\out\server.js:34:2247)
    at wasm://wasm/00470b62:wasm-function[1050]:0xc3f3e
    at wasm://wasm/00470b62:wasm-function[478]:0xb01cb
    at wasm://wasm/00470b62:wasm-function[727]:0xbf160
    at wasm://wasm/00470b62:wasm-function[702]:0xbafce
    at wasm://wasm/00470b62:wasm-function[86]:0x65547
    at wasm://wasm/00470b62:wasm-function[83]:0x608de
    at as.onDocumentFormatting (c:\Users\fomin\.vscode\extensions\rszyma.vscode-kanata-0.8.4\out\server.js:34:812)
    at c:\Users\fomin\.vscode\extensions\rszyma.vscode-kanata-0.8.4\out\server.js:34:6255
    at c:\Users\fomin\.vscode\extensions\rszyma.vscode-kanata-0.8.4\out\server.js:33:77865


[Error - 10:35:31 AM] Request textDocument/formatting failed.
  Message: Request textDocument/formatting failed with message: unreachable
  Code: -32603 
[Error - 10:35:31 AM] Notification handler 'textDocument/didSave' failed with message: recursive use of an object detected which would lead to unsafe aliasing in rust
```

**Comment by rszyma** [comment]
thanks, can reproduce

**Comment by rszyma** [comment]
fixed in v0.9.0
